app/Core/Support/FromFormat.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Leantime\Core\Support;
+
+/**
+ * Enum class FromFormat
+ *
+ * An enumeration class representing various formats for date and time values.
+ */
+enum FromFormat {
+
+    /**
+     * For value containing date time string from database in UTC
+     */
+    case DbDate;
+
+    /**
+     * For value containing both date and time in users preferred format and timezone separated by a space
+     */
+    case UserDateTime;
+
+    /**
+     * for values containing only the user formatted date and timezone. Adds start of day time to string
+     */
+    case UserDateStartOfDay;
```

Comment:
Do we need these start and end of day?

Reply:
They are just a helper for the format() function. It allows us to pass in a string date and then just say start of day in our template files without having to instantiate a DateTimeHelper.